java/arcs/jvm/util/testutil/FakeTime.kt

```diff
@@ -13,9 +13,10 @@ package arcs.jvm.util.testutil
 
 import arcs.core.util.Time
 
-class TimeImpl(val millis: Long? = null) : Time() {
+class FakeTime(var millis: Long = 999_999) : Time() {
     override val currentTimeNanos: Long
-        get() = System.nanoTime()
```

Comment:
System.nanoTime() is not currentTimeNanos, ie they are not nanos from epoch (https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#nanoTime()).

So I think this method should be renamed in Time.kt (or at least the comment, to remove the reference to epoch) otherwise JvmTime does not really implement it correctly.

Reply:
Good call, renamed